tests.py

```diff
@@ -1,107 +1,192 @@
+#!/usr/bin/python
```

Comment:
Unit testing is always a good thing. They are also easy to set up - I am happy to help out. Python even offers '[doctests](https://docs.python.org/2/library/doctest.html)' which can be integrated with Sphix.